src/translate/options.py

```diff
@@ -49,6 +49,10 @@ def parse_args():
         "--keep-unimportant-variables",
         dest="filter_unimportant_vars", action="store_false",
         help="keep variables that do not influence the goal in the causal graph")
+    argparser.add_argument(
+        "--keep-no-ops",
+        dest="keep_no_ops", action="store_true",
+        help="keep operators without effects in the output")
```

Comment:
Minor polishing, but why not: I think we can now remove the "dest" argument because "keep_no_ops" is the default if the option is called "--keep-no-ops". It's a bit magic, but it looks like we exploit the default in the other cases where it works (e.g. "--add-implied-preconditions", "--dump-task".)

Reply:
Makes sense, all set!